docs/migration/2211_ng19/migration.md

```diff
@@ -1,24 +1,37 @@
 # (EARLY NOTES) Migrating a custom app to use Spartacus with Angular v18
 
 Before upgrading Spartacus to the new version with Angular 18, you need to first:
-- upgrade to the latest 2211.x of Spartacus
+
+- upgrade Spartacus to version 2211.32.1
+- install Node 22 version and upgrade `@types/node` to version 22
```

Comment:
Those are 2 different big steps, lets not mix them in one bullet (so customers don't skip one of them unconsciously):
```suggestion
- install Node 22 version
- upgrade package `@types/node` to version 22
```